util.h

```diff
@@ -38,6 +39,7 @@ typedef struct {
   int interactive;
   int cue;
   const char *auth_file;
+  const char *authpending_file;
```

Comment:
need to look into how those are managed now. This is not const anymore. You should get a warning when calling free on it

Reply:
You are right, with `const` I get a couple of warning. I removed the `const`, but a different warning appears:

```
pam-u2f.c: In function ‘parse_cfg’:
pam-u2f.c:58:29: warning: assignment discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
       cfg->authpending_file = argv[i] + 17;
```